### DISCUSSION TITLE
[V2] Overrriding views

### DISCUSSION THREAD

**Discussion by vovacorgo** [awcodes/filament-curator#290] [Q&A] [answered]
Can in curator 2 override views to modifi them for example for changing translate method, If that possible shoud be fine detailed instruction. I tried to copy views to folder `resource/views/vendor/filament-curator`  then components etc...

**Comment by awcodes** [comment]
What are you trying to change? Typically you'd publish the translations and update them in your app. If the plugin is missing any translations or doesn't have a translations directory for your language a PR would be welcomed.

But you shouldn't publish views unless you are willing to except the risk of breaking changes in the future.

**Reply by vovacorgo** [reply]
I have my custom _t() method what uses redis+ database+file translation. That why i need custom view. Can you help me with publish/ use custom views?

**Reply by vovacorgo** [reply]
I am not planning to change much and not planning to update plugin from v2. But i realy need some custom blade files

**Reply by awcodes** [reply, marked as answer]
You should be able to publish them with the normal vendor publish artisan commands. But I will not offer support if you do this.

**Comment by zainabalsharay** [comment]
Yes, I’m not overriding any views. I only use Laravel translation overrides under /lang/vendor/curator/ar and keep all changes limited to translation files to avoid breaking updates.
